lib/event_listeners/foodEvents.js

```diff
@@ -1,70 +1,55 @@
 const $ = require('jquery')
 const Food = require('../models/Food')
 const Ajax = require('../ajax_requests/ajax')
-const HTML = require('../response_handlers/foodHandlers')
+const FoodHandler = require('../response_handlers/foodHandlers')
 
 // event listener for adding food
 
 $(document).ready(function(){
   $("#food_form").submit(event => {
     let name = $("input[name=name]").val()
     let calories = $("input[name=calories]").val()
-    if (name.length === 0) {
-      event.preventDefault();
-      $("p#submit-food-name").append("Please enter a food name.")
-    } else if (calories.length === 0) {
-      event.preventDefault();
-      $("p#submit-calories").append("Please enter a calorie amount.")
-    } else {
-      $("p#submit-food-name").remove()
-      $("p#submit-calories").remove()
-      Ajax.postFoods(name, calories)
-      $('input').val('').removeAttr('checked').removeAttr('selected');
-    }
+    let thisObject = event
+    return FoodHandler.validateFoodInput(name, calories, thisObject)
   })
 })
 
 // event listener for deleting food
 
-$("#food-list").click(event => {
-  let objectId = event.target.parentElement.parentElement.dataset.id
-  let row = event.target.parentElement.parentElement.parentElement
-  Ajax.deleteFood(objectId, row)
+
+$(document).ready(function(){
+  $("#food-list").click(event => {
+    if ((event.target.className === "delete-button") || (event.target.parentElement.className === "delete-button")) {
+    let objectId = event.target.parentElement.parentElement.dataset.id
```

Comment:
I still think this is crazy how this is much faster than calling .parent(). dang built in methods